css-pseudo-4/Overview.bs

```diff
@@ -696,6 +696,8 @@ Styling Highlights</h3>
     <wpt>
     css/css-pseudo/textpath-selection-011.html
     </wpt>
+
+    <li>A [=custom property=] may be defined and/or used inside a highlight pseudo-element.
```

Comment:
```suggestion
    <li>[=custom properties=] [[!CSS-VARIABLES-1]]
```

A property applying means you can declare or use it, custom properties aren't different from others in this respect.